### ISSUE TITLE
lubridate::here

### ISSUE THREAD

**Issue by tyner** [hadley/r-pkgs#1033]
In section 10.2.1 of this,

https://r-pkgs.org/dependencies-mindset-background.html

it says "both the lubridate and here packages provide a here() function" however I am not seeing `lubridate::here` at all. Was it perhaps removed?

**Comment by jennybc** [comment]
Yeah, it was removed. So this chapter should feature a different example. Would you like to do one of these PRs?

* Add a footnote explaining that `here()` has since been removed from lubridate. A stopgap, but better than nothing.
* Pick a different pair of functions and update the example.
